frontend/src/metabase/core/components/AccordionList/AccordionListCell.styled.tsx

```diff
@@ -10,10 +10,6 @@ export const ListCellItem = styled.div<ListCellItemProps>`
   border-color: ${props => props.isClickable && alpha("accent2", 0.2)};
 `;
 
-export const FilterContainer = styled.div`
-  padding: 0.5rem;
-`;
```

Comment:
I guess the `ListSearchField` itself has enough padding because of the Mantine text input, so we don't need a filter container?

Reply:
yeah, I added it directly to the component as a mantine style prop rather than yet another div with a single property